tasks/grunt-aws-cloudformation.js

```diff
@@ -109,6 +109,16 @@ module.exports = function(grunt) {
 					}
 				});
 				break;
+			case "stack-status":
+				var statusParams = {
+					StackName: data.stackName,
+					NextToken: data.nextToken
+				};
+				cloudformation.describeStacks(statusParams, function(err, data) {
+					if (err) grunt.warn(err, err.stack);
+					else     grunt.log.writeln("Stack status:\n" + JSON.stringify(data, null, 1));
```

Comment:
Since describeStacks is async, then the callback should use the `done()` method like:
```
cloudformation.describeStacks(statusParams, function(err, data) {
   if (err) {
      grunt.warn(err, err.stack);
   } else {
      grunt.log.writeln("Stack status:\n" + JSON.stringify(data, null, 1));
      done();
   }
```
otherwise any tasks following this one won't execute properly.  I'll go ahead and merge your PR and make that update myself, then publish the new package. Thanks for the help on these new actions!